src/main/java/io/kontur/disasterninja/dto/EventListDto.java

```diff
@@ -23,5 +23,7 @@ public class EventListDto {
     private List<Double> bbox;
     private List<Double> centroid;
     private int episodeCount;
+    private String eventType;
+    private OffsetDateTime startedAt;
```

Comment:
I'd move `startedAt` to `updatedAt` and `eventType`  between `eventId` and `eventName` not to look for these fields through the whole class next time.